packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -374,20 +364,17 @@ describe('plugin-devices', () => {
         });
 
         describe('when the device is not registered', () => {
-          beforeEach('check that the device is not registered', () => {
-            assert.isFalse(devices.registered);
-          });
-
           describe('when the device successfully registers', () => {
-            it('should resolve the promise with the websocket url', () => Promise.all([
-              devices.getWebSocketUrl(wait),
-              devices.register()
-            ])
-              .then(([url]) => {
-                assert.isDefined(url);
-                assert.isTrue(services.isServiceUrl(url));
-                assert.include(url, 'mercury');
-              }));
+            it('should resolve the promise with the websocket url',
+              () => Promise.all([
+                devices.getWebSocketUrl(wait),
+                devices.register()
+              ])
+                .then(([url]) => {
+                  assert.isDefined(url);
+                  assert.isTrue(services.isServiceUrl(url));
+                  assert.include(url, 'mercury');
```

Comment:
Is there a better way to verify that this is a websocket url instead of just checking for the "mercury" string?

Reply:
There is an improved way to do this, I will amend it now.